src/Elewant/AppBundle/Resources/translations/herd.en.yml

```diff
@@ -61,7 +61,8 @@ breed:
   WHITE_DPC_REGULAR: "DPC"
   WHITE_GLOBALIS_REGULAR: "Globalis"
   WHITE_ZEND_REGULAR: "Zend ElPHPis"
-  YELLOW_SUNSHINEPHP_REGULAR: "SunshinePHP"
+  YELLOW_SUNSHINEPHP_REGULAR: "SunshinePHP (Zend)"
```

Comment:
Maybe also change the name for the `Large` version of this ElePHPant?

Reply:
Good catch!